test/settings/SettingsStore-test.ts

```diff
@@ -31,7 +31,7 @@ const TEST_DATA = [
 /**
  * An existing setting that has {@link IBaseSetting#supportedLevelsAreOrdered} set to true.
  */
-const SETTING_NAME_WITH_CONFIG_OVERRIDE = "feature_new_room_decoration_ui";
+const SETTING_NAME_WITH_CONFIG_OVERRIDE = "feature_msc3531_hide_messages_pending_moderation";
```

Comment:
Why is it needed in this PR?

Reply:
Because `feature_new_room_decoration_ui` no longer has a config override, so this test fails due to it relying on a setting which no longer matches its expectations. It really should be making its own setting but that was a larger change than necessary here